### PR TITLE
Githubactions vpc

### PR DESCRIPTION
**Pull request by NishaAnilBidave**
final githubactions file.

### CODE REVIEW

File: .github/workflows/group-3-vpc.code-workspace [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
```
Comment (pasc-ed) [inline .github/workflows/group-3-vpc.code-workspace]: Could you remove this file please, I should not be pushed to git.

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+name: VPC deployment in AWS
+on:
+  push:
+    branches:
+      githubactions_vpc
```
Comment (pasc-ed) [inline .github/workflows/main.yml]: This branch will not exists after the merge. So it should set to main branch or any other branches instead.

File: backend.tf [hunk 3]
```diff
@@ -0,0 +1,48 @@
+terraform {
+  backend "s3" {
+    bucket         = "talent-academy-pascal-lab-tfstates"
+    key            = "talent-academy/vpc_group3/terraform.tfstates"
+    region         = "eu-west-1"
+    dynamodb_table = "terraform-lock"
+  }
+}
+
+# resource "aws_iam_openid_connect_provider" "default" {
```
Comment (pasc-ed) [inline backend.tf]: These commented lines should be removed completely.

**Review by pasc-ed — CHANGES_REQUESTED** [review]
I've left a few comments throughout the files. Could you solve these issues first and we'll be able to merge. Thanks

File: backend.tf (hunk 3 above)
Comment (NishaAnilBidave) [reply backend.tf]: This is done.

File: .github/workflows/main.yml (hunk 2 above)
Comment (NishaAnilBidave) [reply .github/workflows/main.yml]: Branch name has been changed to main.

File: .github/workflows/group-3-vpc.code-workspace (hunk 1 above)
Comment (NishaAnilBidave) [reply .github/workflows/group-3-vpc.code-workspace]: Removed the file.

**Review by pasc-ed — APPROVED** [review]
LGTM

**Comment by NishaAnilBidave** [comment]
Merge approved by Pascal.

**Pull request merged**